var/spack/repos/builtin/packages/bigdft-atlab/package.py

```diff
@@ -38,7 +39,7 @@ class BigdftAtlab(AutotoolsPackage):
     depends_on("mpi", when="+mpi")
     depends_on("openbabel", when="+openbabel")
 
-    for vers in ["1.9.0", "1.9.1", "1.9.2", "1.9.3", "1.9.4", "develop"]:
+    for vers in ["1.9.0", "1.9.1", "1.9.2", "1.9.3", "1.9.4", "1.9.5", "develop"]:
         depends_on(f"bigdft-futile@{vers}", when=f"@{vers}")
 
```

Comment:
Could we reduce the complexity by maybe specifying it this way:
```suggestion
    with when("@1.9.0:"):
        vers = self.version
        depends_on(f"bigdft-futile@{vers}")

```
I haven't tested this out but seems plausible.

Reply:
Should I create an `__init__` for the class, such that we have `self` in scope?

Reply:
@viperML:

I think it would be best to use the regular standard method to add new dependencies:
```py
depends_on("bigdft-futile@1.9:")
depends_on("bigdft-futile@develop")  # (only if needed when @1.9.0: does not include develop)
```

@viperML Please ping me or other `spack` maintainers by mentioning us to get your change reviewed again.

@viperML: Also, I recommend submit a tiny PR that only updates a base dependency (e.g. like `libgain` or `ntpolt` - as long as these update don't depend on each other) in new PRs to get on top of the review queue and make approval and fixes easier than with a big PR like this.